src/lnurlp.rs

```diff
@@ -49,6 +49,7 @@ pub async fn well_known_lnurlp(
 
 const MAX_AMOUNT: u64 = 100_000_000 * 1_000; // 1 BTC
 const MIN_AMOUNT: u64 = 5_000; // 5 sats
+const MAX_COMMENT_LEN: i32 = 100;
```

Comment:
should just make this a `usize` so you don't have to cast it later

Reply:
had it as `i32` because I'm also using it here `comment_allowed: Some(MAX_COMMENT_LEN)` and this `comment_allowed` field is `i32` so would need cast one or the other

Reply:
ah okay, nvm